### PR TITLE
#24702 org mapping

### PR DESCRIPTION
**Pull request by danchr**
https://redmine.magenta-aps.dk/issues/24702

- [x] Have you updated the documentation?
- [x] Have you performed a smoke test of the application?
- [x] Have you written tests for your changes?
- [x] Have you updated the release notes?

ss-01-frontpage
ss-02-mapper-initial
ss-03-mapper-selected

### CODE REVIEW

File: backend/tests/mocking/update-fixture.py [hunk 1]
```diff
@@ -11,41 +11,49 @@
 Update, i.e. re-fetch, a JSON fixture
 '''
 
+import json
 import operator
 import os
-import json
+import pathlib
```
Comment (cmoesgaard) [inline backend/tests/mocking/update-fixture.py]: `pathlib` (and `operator`) is not used.

File: backend/tests/test_integration_org_unit.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
+import operator
```
Comment (cmoesgaard) [inline backend/tests/test_integration_org_unit.py]: Unused import

File: frontend/src/components/MoTreeView/MoTreeView.vue [hunk 3]
```diff
@@ -189,31 +229,75 @@ export default {
   },
 
   methods: {
+    onNodeCheckedChanged (node) {
+      if (this.multiple) {
+        let checked = this.getSelection()
+
+        console.log(`TREE: checked ${checked.join(', ')}`)
```
Comment (n-singh) [inline frontend/src/components/MoTreeView/MoTreeView.vue]: Remove console log :)

File: frontend/src/components/MoTreeView/MoTreeView.vue [hunk 4]
```diff
@@ -189,31 +229,75 @@ export default {
...
      *
      * @protected
      */
     onNodeSelected (node) {
-      /**
-       * Emitted whenever the selection changes.
-       */
-      this.$emit('input', node.id)
+      if (!this.multiple) {
+        console.log(`TREE: selected ${node.id}`)
```
Comment (n-singh) [inline frontend/src/components/MoTreeView/MoTreeView.vue]: Same here

File: frontend/src/components/MoTreeView/MoTreeView.vue [hunk 5]
```diff
@@ -189,31 +229,75 @@ export default {
...
+     * they're present, and updating the tree otherwise.
      */
-    setSelection (unitid) {
-      unitid = unitid || this.value
+    setSelection (unitids) {
+      // wrap the values in a list, if necessary, handling absence
+      const newVal = this.toArray(unitids)
+      const oldVal = this.getSelection()
+
+      console.log(`TREE: set selection from ${oldVal} to ${newVal}`)
```
Comment (n-singh) [inline frontend/src/components/MoTreeView/MoTreeView.vue]: And here

File: frontend/src/modules/organisationMapper/_store/index.js [hunk 6]
```diff
@@ -1,29 +1,62 @@
...
+    Service.post(`/ou/${state.origin}/map`,
+                 {
+                   destination: state.destination,
+                   validity: {
+                     from: moment().format('YYYY-MM-DD')
+                   }
+                 },
+                )
+      .then(response => {
+        console.log('submitted, everything is great')
```
Comment (n-singh) [inline frontend/src/modules/organisationMapper/_store/index.js]: Maybe here too

File: frontend/src/modules/organisationMapper/index.vue [hunk 7]
```diff
@@ -1,27 +1,101 @@
...
-    const STORE_KEY = '$_chat'
 
-    if (!(STORE_KEY in this.$store._modules.root._children)) {
-      // this.$store.registerModule(STORE_KEY, store)
+  watch: {
+    origin (newVal) {
+      this.$store.dispatch('organisationMapper/GET_ORGANISATION_MAPPINGS')
+    },
+    destination (newVal) {
+      console.log('destination changed!')
```
Comment (n-singh) [inline frontend/src/modules/organisationMapper/index.vue]: And here :)

**Pull request merged**